### PR TITLE
Turkish translation

### PR DESCRIPTION
**Pull request by realbaycan**
Turkish language translation, consisting of 177 words

### CODE REVIEW

File: ch07-git-tools.asc [hunk 1]
```diff
@@ -1,10 +1,10 @@
 [[ch07-git-tools]]
-== Git Tools
+== Git Araçları
 
-By now, you’ve learned most of the day-to-day commands and workflows that you need to manage or maintain a Git repository for your source code control.
-You’ve accomplished the basic tasks of tracking and committing files, and you’ve harnessed the power of the staging area and lightweight topic branching and merging.
+Şu ana kadar, kaynak kodu denetimi için bir Git deposu yönetmek veya korumak için ihtiyaç duyduğunuz günlük komut veiş akışlarının çoğunu öğrendiniz.
```
Comment (gokaygurcan) [inline ch07-git-tools.asc]: "komut ve iş akışları"

**Review by gokaygurcan — CHANGES_REQUESTED** [review]

**Comment by realbaycan** [comment]
I updated. Thank you.

**Review by gokaygurcan — APPROVED** [review]

**Comment by jnavila** [comment]
Hi @gokaygurcan 
You approved the changes but did not merge them. I'd like to know if you need the requested privileges to do so.

BTW, at some point in the future, I would like to set up the automatic compilation of the book to pdf and ebook on this project (for download from git-scm.com). Let me know when the project is calm, so that I can make the necessary technical changes.

**Comment by gokaygurcan** [comment]
Hi @jnavila,

After almost 3 years, people started to translate the book. Probably that's because of https://utopian.io/ and its rewarding system. But anyways, in the end we'll have a finished book and i care about that part most. 

The thing about this translations, mostly these people are using Google translate (not talking particularly about this PR or user, it's the general approach of the users of that platform) and it's not working 100% when you translate from En to Tr. I need to read all of it, make sure it's correct then accept the changes in an order. Right now i'm in reading phase and probably in couple of days they'll be merged back to the repository.

That'd be great to setup automated processes when we have at least some sections translated. If you need any help, you can just ping me.

**Pull request merged**